assets/src/edit-story/app/media/local/useContextValueProvider.js

```diff
@@ -72,11 +72,13 @@ export default function useContextValueProvider(reducerState, reducerActions) {
         .then(({ data, headers }) => {
           const totalPages = parseInt(headers.get('X-WP-TotalPages'));
           const mediaArray = data.map(getResourceFromAttachment);
+          const hasMore = p < totalPages;
           callback({
             media: mediaArray,
             mediaType: currentMediaType,
             searchTerm: currentSearchTerm,
             pageToken: p,
+            nextPageToken: hasMore ? p + 1 : undefined,
```

Comment:
Can we add a todo / comment that this is just temporary and needs to be changed?

Reply:
Today, the page token for local uploaded media is the page number.
What change did you have in mind?

Reply:
Ah so for the local it will always stay like that. Then it's fine.